kin-core/src/main/java/kin/core/exception/TransactionFailedException.java

```diff
@@ -1,10 +1,13 @@
-package kin.sdk.core.exception;
+package kin.core.exception;
 
 
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import java.util.List;
 
+/**
+ * Blockchain transaction failure has happened, contains blockchain specific error details
+ */
```

Comment:
note the `opResultCode` below, the transaction failure can have several operation codes